scene-format.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+  message Object {
+    int32 id;
+    string reference;         // path to file or url
+  }
+  message Material {
+    int32 object_id;
+    string type;
+  }
+  message Light {
+    int32 id;
+    string type;
+    repeated double position; // array of three numbers
+  }
+  message Camera {
+    int32 id;
+    repeated double position;
+  }
+  repeated Object objects;
+  repeated Material materials;
+  repeated Light lights;
+  repeated Camera cameras;
```

Comment:
What about screen resolution and from which camera we should render?

Reply:
I don't think we need those. It was mentioned somewhere that these parameters are specified using command-line arguments with some default values as a fallback. Need to double-check.
On the other hand, specifying those fallback values in a file could probably make sense. What do you think?

Reply:
I check assignment 3_1 and there is no mention of this. I think it is more practical to just add these parameters to the file instead of using command-line arguments

Reply:
Apparently, it was in the previous version of the file and got removed later:
```
#### Як має працювати рендер
Вимоги прості: на вхід приймається файл затеврдженого формату, аргументом передається номер камери (або id) всередині сцени, з погляду якої має відбуватись рендер та параметри зображення (ширина\висота).
```

Let's add those fields anyway, and it will be up to users to override them using command line args.